content/blog/security-and-product-teams/index.md

```diff
@@ -0,0 +1,76 @@
+ 
+**Do not assume** you know what your core audience needs, doing some customer research early on goes a long way to prevent the product from focusing on the wrong things.
+ 
+## Minimum Viable Product (MVP)
+ 
+After identifying the value proposition, what is the minimum set of features you can provide your customers? The faster the product can validate that their value proposition and market segment are correct the higher the chances of that product succeeding. Product teams often prototype something quick and throw it away once their value proposition is validated. AppSec teams can employ the same technique, but they should be careful to manage stakeholder expectations in order to not lose trust. Cutting corners without a clear action plan has bigger consequences for AppSec.
+ 
+## User Experience
+ 
+I will loosely use the term user experience as the ability for customers to use the product independently of where in their journey they are. As a new customer adopting the product, how much work is expected from them to get started? Do they need to refer to documentation or reach out in another channel (Slack channels, email, phone, etc) for support? Can someone experienced with the product customize it to better fit their needs? These should all be conscious decisions before releasing the product.
+ 
+As a rule of thumb the easier the product is from the get-go ("no config" and good defaults) the more it will gain traction. Allowing it to be customized _after_ customers are familiar with it will create a loyal customer base and increase customer retention. In other words, try to defer complexity to the later stages of the customer journey.
+ 
+Before we finish this section we should mention "breaking changes". One of the main reasons for customer retention are customers feeling that they have mastered your product. Shipping breaking changes force customers to go back on their learning journey and give them the perfect opportunity to consider easier alternatives. How many times have you stuck with a product despite a modern one existing just because you know it end to end?
+ 
+## Data
+ 
+Data is a broad topic and has two sides here:
+ 
+1. Data used to drive the product
+2. Data provided to customers through product usage
+ 
+### Data used to drive the product
+ 
+Are decisions to add, modify or remove a feature backed by data? When a new version of the product is released, can adoption and feature usage be measured? Is customer feedback being collected? All of these questions can help the product be successful and stay relevant for the long term. This will also enable continuous experimentation within the product.
+ 
+Pay attention to your early adopters and product advocates' feedback, they are a driving force that can help the product be adopted by a larger audience.
```

Comment:
One thing that may be worth adding here: data analysis is non1useful without context. Junk-in leads to junk-out: if you don't have clear goals on what your desired outcomes are, no about of data will save you.

This article is about product teams so it may be worth touching on how product teams decide what to build. In my experience, this usually starts by identifying a problem or undesirable state and making a plan (which may include one or more features) to reach a set of desired outcomes.

You can measure things before you have desired outcomes, but you're more likely to measure things that don't matter if you don't have clear questions to answer.